nncf/torch/pruning/export_helpers.py

```diff
@@ -488,13 +571,19 @@ def __init__(self, model: NNCFNetwork, graph: NNCFGraph,
                  pruning_operator_metatypes: PruningOperationsMetatypeRegistry):
         super().__init__(graph, pruning_operator_metatypes)
         self._model = model
+        self._is_mask_propogated = False
 
     def apply_mask(self):
         """
         Applying propagated masks for all nodes in topological order:
         1. running input_prune method for this node
         2. running output_prune method for this node
         """
+        if not self._is_mask_propogated:
+            nncf_logger.warning('Mask propogation didn\'t called before'
+                                ' apply_mask call in ModelPruner. This can lead to'
+                                ' incorrect model pruning export.')
+
```

Comment:
@vshampor, shall we just inform user or raise runtime error? I would preferer runtime error

Reply:
`propogation` -> `propagation`. RuntimeError is more suitable here.